camelot/model/type_and_status.py

```diff
@@ -322,6 +339,7 @@ def get_state(self, model_context):
                 'current_status'
             )
             current_status = obj.current_status
+            state.color = status_colors[current_status]
```

Comment:
Ik zag net dat dit bestand niet meer bestaat in master...?

Reply:
nvm, blijkbaar verhuisd naar vfinance/model/bank/status.py

Reply:
ja, ik ben alle model code ah verhuizen naar vf, zodat we nt meer met die back and forth imports en patches zitten